test/fuzz.js

```diff
@@ -53,7 +53,7 @@ describe('fuzzing', () => {
   }).timeout(5000)
 
   it('should handle random strings', () => {
-    for (let i = 0; i < 5000; ++i) {
+    for (let i = 0; i < 1000; ++i) {
```

Comment:
Is the `.timeout(5000)` added in the previous PR (line 64) necessary reducing it from 5000 => 1000 ?

Reply:
I've set the timeout to 5000 but it was still timing out on alpine linux some times, so instead of having flaky-ish tests I'd figure i'd just lower the iterations yes